eng/Versions.props

```diff
@@ -95,6 +95,8 @@
     <MicrosoftVisualStudioComponentModelHostVersion>16.0.198-g52de9c2988</MicrosoftVisualStudioComponentModelHostVersion>
     <MicrosoftVisualStudioCompositionVersion>15.5.23</MicrosoftVisualStudioCompositionVersion>
     <MicrosoftVisualStudioCoreUtilityVersion>16.0.203-g5d5c63964e</MicrosoftVisualStudioCoreUtilityVersion>
+    <MicrosoftVisualStudioDebuggerEngineVersion>15.9.28307</MicrosoftVisualStudioDebuggerEngineVersion>
+    <MicrosoftVisualStudioDebuggerMetadataVersion>15.9.28307</MicrosoftVisualStudioDebuggerMetadataVersion>
```

Comment:
These are not used. We need to update `MicrosoftVisualStudioDebuggerEngineimplementationVersion` and `MicrosoftVisualStudioDebuggerMetadataimplementationVersion` instead.

Reply:
I wonder why do the test pass :-/

Reply:
The packages are in https://www.myget.org/F/roslyn_concord/api/v3/index.json, but the last version there is 15.7.